textpipe/doc.py

```diff
@@ -274,3 +275,20 @@ def _set_default_nlp(self, lang):
             self._spacy_nlps[lang] = {}
         model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
         self._spacy_nlps[lang][None] = model
+
+    @property
+    def sentiment(self):
+        """
+        Returns sentiment score (-1 to 1) and a confidence level (0 to 1)
+
+        Currently only Dutch supported
+
+        >>> doc = Doc('Een leuke test zin.')
+        >>> doc.sentiment
+        (0.6, 0.9666666666666667)
+        """
+
+        if self.language == 'nl':
+            return sentiment(self.clean_text())
```

Comment:
I think we should use `self.clean` instead.

Reply:
agree, although I see that in _load_spacy_doc clean_text() is used as well..